.github/TEST_MATRIX.md

```diff
@@ -15,6 +15,7 @@ Molecule unit tests
 |![core/firewall unit testing](https://github.com/bluebanquise/bluebanquise/workflows/core/firewall%20unit%20testing/badge.svg)                       | x | x |   |
 |![core/log_client unit testing](https://github.com/bluebanquise/bluebanquise/workflows/core/log_client%20unit%20testing/badge.svg)                   | x | x | x |
 |![core/log_server unit testing](https://github.com/bluebanquise/bluebanquise/workflows/core/log_server%20unit%20testing/badge.svg)                   | x | x | x |
+|![core/pxe_stack unit testing](https://github.com/bluebanquise/bluebanquise/workflows/core/pxe_stack%20unit%20testing/badge.svg)                   | x | x |   |
```

Comment:
Can you try to keep the columns aligned? It's not always possible with very long lines, but it helps readability.